pkg/scaffold/v1/webhook/webhook_test.go

```diff
@@ -20,7 +20,6 @@ var _ = Describe("Webhook", func() {
 	}
 
 	serverName := "default"
-	domainName := "testproject.org"
```

Comment:
Why you remove the tests?

Reply:
That is checked in resource, this hard-coded string was used to fill `GroupDomain` which is one of the removed fields.

Reply:
Did you find it has other place testing and ensuring that the webhooks after this change will still work on with domains as `"testproject.org"`? If yes, could you please provide the info in the PR? I mean add the link where it still been checked. If not, then is required to add the test. 

Also, did you check it manually?

Reply:
The test just filled that value with an string that was never used. You can try changing that string for a invalid domain and it will still work, as it was not being used for anything.

You can see the removal of that field in [here](https://github.com/kubernetes-sigs/kubebuilder/pull/1219/files/8c42e7ef0628965330882d9f02ec377ec1f732a8#diff-4b3b663bcd342da03284c5e58972d498) and [here](https://github.com/kubernetes-sigs/kubebuilder/pull/1219/files/8c42e7ef0628965330882d9f02ec377ec1f732a8#diff-ae6f681b954c192fb1fb62fdb59e7091). Doing a search on those files (not in the changes section of the PR but in the real file) shows that `GroupDomain` is only used to declare it and assign a value, but its value is never used in the scaffolded file.